components/css/global/_base.scss

```diff
@@ -10,77 +10,66 @@ body{
 	body{
 		font-family:$font-arial;
 		font-size:1em;
-		line-height:24/16*1em;
+		line-height:calc-toEm(24);
 	}
 }
 @media (max-width:767px){
 	body{
-		font-size:18/16*1em;
-		line-height:24/18*1em;
+		font-size:calc-toEm(18);
+		line-height:calc-toEm(24);
 	}
 }
 h1{
 	padding-top:0;
 	padding-bottom:0.25em;

```

Comment:
I made this mistake when I originally setup this file, but I think these should be margins instead of padding. Since margins collapse on each other, it would be easier to manage vertically spacing elements instead of handling it via more complex selectors, such as `h1+h2`. We'll have to be cautious that implementing margins instead of padding doesn't affect templates.

Reply:
Yes, I wondered about that and I agree. I'll comment these sections out and switch to margins and test.